cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/bridge_to_ethereum_config.rs

```diff
@@ -56,9 +59,18 @@ pub type SnowbridgeExporter = EthereumBlobExporter<
 >;
 use hex_literal::hex;
 
+pub type SnowbridgeExporterV2 = EthereumBlobExporterV2<
+	UniversalLocation,
+	EthereumNetwork,
+	snowbridge_pallet_outbound_queue_v2::Pallet<Runtime>,
+	snowbridge_core::AgentIdOf,
+	EthereumSystemV2,
+	AssetHubParaId,
+>;
+
 // Ethereum Bridge
 parameter_types! {
-	pub storage EthereumGatewayAddress: H160 = H160(hex!("EDa338E4dC46038493b885327842fD3E301CaB39"));
+	pub storage EthereumGatewayAddress: H160 = H160(hex!("b1185ede04202fe62d38f5db72f71e38ff3e8305"));
```

Comment:
Very easy to slip a change here with disastrous effects. For extra safety, please add a runtime test along the lines:

```rust
fn sanity_check_snowbridge_gateway() {
	// you can validate this at https://etherscan.io/address/0x4a4559CCD9195C3CABBd4Da00854A434E8dd2Ea3
	let expected_gateway = "0x4a4559CCD9195C3CABBd4Da00854A434E8dd2Ea3";
	assert_eq!(EthereumGatewayAddress::get(), expected_gateway)`;
}
```

Makes any future changes to the gateway more visible.

Reply:
@yrong I'm assuming `b1185ede04202fe62d38f5db72f71e38ff3e8305` is the deployment of V2 gateway on Sepolia?

Reply:
It's V2 Gateway for local setup, though we have initial script to change it.

https://github.com/Snowfork/snowbridge/blob/e4af25d9a2a8cfc52245528e0ed3dbab6e046364/web/packages/test/scripts/configure-substrate.sh#L77-L83

It's changable, so I don't think it's important to add an assert here. e.g., for V2 gateway on Sepolia, I asked help from Westend DevOps to set that.

Reply:
yes, not important for Westend but important for Polkadot, adding test here and there keeps things consistent